Bottom: c2cb5827d5058d1c60153dfc5585a8f00476bbc3
Top:    3c51c4d4cf7e766f737afc6a275d64bb41b12314
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-01-22 17:07:06 +0800

Refresh of virtio-net-add-missing

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 12dfc5f..4e5a663 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -1222,8 +1222,10 @@ static int virtnet_poll(struct napi_struct *napi, int budget)
 	if (received < budget)
 		virtqueue_napi_complete(napi, rq->vq, received);
 
-	if (xdp_xmit)
+	if (xdp_xmit) {
+		virtqueue_kick(rq->vq);
 		xdp_do_flush_map();
+	}
 
 	return received;
 }
